scripts/badger/claim_aura_rewards.py

```diff
@@ -0,0 +1,40 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main(
+    msig_name="treasury_voter_multisig",
+    redirect_rewards=False,
+    destination="0xA9ed98B5Fb8428d68664f3C5027c62A10d45826b",  # defaults to voter_msig
+):
```

Comment:
this is super confusing, what is the purpose of this script?

Reply:
it is a generalised script for claiming rewards from aura. allows claiming from different msigs and as usual, if we need to redirect rewards to voter, we can do it in one go within the same script